src/libproxychains.c

```diff
@@ -479,3 +481,18 @@ struct hostent *gethostbyaddr(const void *addr, socklen_t len, int type) {
 	}
 	return NULL;
 }
+
+ssize_t sendto(int sockfd, const void *buf, size_t len, int flags,
+	       const struct sockaddr *dest_addr, socklen_t addrlen) {
+#ifdef MSG_FASTOPEN
```

Comment:
Since we don't actually call sendto(FASTOPEN) on old kernels that do not support it, but issuing connect, this seems to be a good idea. I will try to test it then write back to you. Thanks for suggestion.